src/extension/background-script/nostr/index.ts

```diff
@@ -16,23 +17,24 @@ class Nostr {
     return null;
   }
 
-  getPublicKey() {
+  async getPublicKey() {
     const publicKey = secp256k1.schnorr.getPublicKey(
-      secp256k1.utils.hexToBytes(this.getPrivateKey())
+      secp256k1.utils.hexToBytes(await this.getPrivateKey())
```

Comment:
for me to learn: is this a good practice to use `await something` as function parameter? 
(back in the days this was problematic due to polyfills)

Reply:
fixed a think, tested, seems to work